commands/channel.go

```diff
@@ -573,3 +586,42 @@ func getPrivateChannels(c client.Client, teamID string) ([]*model.Channel, *mode
 	}
 	return privateChannels, nil
 }
+
+func getChannelDeleteConfirmation() error {
+	var confirm string
+	fmt.Println("Have you performed a database backup? (YES/NO): ")
+	fmt.Scanln(&confirm)
+
+	if confirm != "YES" {
+		return errors.New("aborted: You did not answer YES exactly, in all capitals")
+	}
+	fmt.Println("Are you sure you want to delete the channels specified? All data will be permanently deleted? (YES/NO): ")
```

Comment:
Would it make it more smooth to have only one confirmation here, and state both permanent deletion and emphasizing the DB backup?

Reply:
I'm open to changing the confirmation flow. We use similar confirmation text for user and team deletion as well.

Reply:
If we decide to change this, the change should be propagated to keep all the confirmation flows consistent